scripts/fast-reboot

```diff
@@ -163,27 +164,17 @@ case "$REBOOT_TYPE" in
             REBOOT_TYPE="fastfast-reboot"
```

Comment:
>REBOOT_TYPE="fastfast-reboot" [](start = 12, length = 29)

We'd better treat fastfast-reboot as Mellanox implementation of warm-reboot, and keep REBOOT_TYPE as warm-reboot.

Reply:
it is a different flow and i think we should keep it different just in case warmboot will have flows which we should not use.
I prefer keeping it that way